src/wbase/Task.cc

```diff
@@ -217,16 +235,11 @@ wpublish::QueryStatistics::Ptr Task::getQueryStats() const {
 }
 
 /// @return the chunkId for this task. If the task has no chunkId, return -1.
-int Task::getChunkId() const {
-    if (msg->has_chunkid()) {
-        return msg->chunkid();
-    }
-    return -1;
-}
+int Task::getChunkId() const { return _chunkId; }
 
 /// Flag the Task as cancelled, try to stop the SQL query, and try to remove it from the schedule.
 void Task::cancel() {
-    LOGS(_log, LOG_LVL_INFO, "Task::cancel " << _idStr);
+    LOGS(_log, LOG_LVL_INFO, "Task::cancel " << getIdStr());
```

Comment:
Perhaps, `LOG_LVL_WARN` would be better in this context. Task cancellation is usually an abnormal, though, not a catastrophic event that might require a bit more attention when inspecting log streams.

Reply:
Maybe, the problem is when one `Task` is cancelled thousands of `Task`s tend to get cancelled and it just fills up the log, quite possibly burrying the thing that matters.